clients/cloud/nodejs/producer.js

```diff
@@ -36,14 +32,16 @@ async function produceExample() {
 
   const producer = await createProducer(config);
 
-  range(10).forEach((count) => {
+  const range = Array(10).keys();
```

Comment:
This is still instantiating an array for iterating over a fixed range of integers, I don't really understand why would you want to do that? Why not just use a for loop?

Reply:
it's creating an empty array of length 10 and then creating another array of length 10 with the range values in it. this (lunacy) is in fact idiomatic javascript (and so probably optimized away nicely), but i wouldn't do this here since it's then just being used in a for loop. I'd only do this if I wanted to `map` over it or something. writing `for (let i=0; i<10; ++i)` is also terser and more straightforward.